datamodel_code_generator/__init__.py

```diff
@@ -500,15 +500,18 @@ def get_header_and_first_line(csv_file: IO[str]) -> Dict[str, Any]:
         header += f'\n#   version:   {get_version()}'
 
     file: Optional[IO[Any]]
-    for path, body_and_filename in modules.items():
-        body, filename = body_and_filename
+    for path, (body, filename) in modules.items():
         if path is None:
             file = None
         else:
             if not path.parent.exists():
                 path.parent.mkdir(parents=True)
             file = path.open('wt', encoding=encoding)
 
+        if custom_file_header and os.path.exists(custom_file_header):
+            with open(custom_file_header) as header_file:
+                custom_file_header = ''.join(header_file.readlines())
```

Comment:
@koxudaxi I took the liberty and added the option to read the header from file as well. We need to add the ASF header to each of the file in Iceberg: https://github.com/apache/iceberg/pull/7751/files Passing in a multi line string as an argument is a bit of a pain, so I figured this would be a nice addition.